packages/dds/sequence/src/test/intervalCollection.spec.ts

```diff
@@ -212,7 +212,7 @@ describe("SharedString interval collections", () => {
 		//
 		// this test involves a crash inside RBTree when multiple intervals slide
 		// off the string
-		it.skip("passes regression test for #4477", () => {
+		it("passes regression test for #4477", () => {
```

Comment:
could you update the description to be a bit more informative without requiring access to the internal item? Probably sufficient to mention historical context that we didn't used to clear a local reference's associated segment/offset upon detaching it, which caused issues in interval collection.